shell/platform/fuchsia/flutter/tests/fakes/scenic/fake_flatland.cc

```diff
@@ -234,6 +234,20 @@ void FakeFlatland::SetOrientation(
   transform->orientation = orientation;
 }
 
+void FakeFlatland::SetOpacity(
+    fuchsia::ui::composition::TransformId transform_id,
+    float value) {
+  if (transform_id.value == 0) {
+    // TODO(fxb/85619): Raise a FlatlandError here
+    FML_CHECK(false) << "FakeFlatland::SetOpacity: TransformId 0 is invalid.";
+    return;
+  }
+
+  if (value < 0.f || value > 1.f) {
+    FML_CHECK(false) << "FakeFlatland::SetOpacity: Invalid opacity value.";
+  }
+}
+
```

Comment:
We should add one for SetScale() as well.

Also, these are mostly added so that we can add it to the FakeGraph and check against it in flatland_external_view_embedder_unittests. I think it might have to trickle down to every content under this transform in the current setup for Opacity. It would be good to cover this behavior in that unittest.

Reply:
I see, I've updated the fakegraph checks to account for the new changes